File-Formats/PDF-to-image/Referring-PDFium.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Referring PDFium | Syncfusion
+description: Learn about Converting PDF pages into images using Syncfusion PdfToImageConverter with the help of PDFium.
+platform: file-formats
+control: PDF to image
+documentation: UG
+---
+
+# Convert PDF to image using PDFium
+
+PdfToImageConverter use PDFium to convert pdf document as image. PDFium is used in Google Chrome for rendering PDF files. It provides accurate and robust PDF rendering. 
+
+N> PDFium is not supported in Windows XP operating system.
+
+### How PDFium works with PdfToImageConverter
+
+* On running your application, Syncfusion PdfToImageConverter control generates a folder named `PDFium` in the application output path folder (for example: bin/release or bin/debug) at runtime. 
+* Syncfusion PdfToImageConverter control detects the architecture of the running machine automatically.
+* Next, it creates another subfolder named “x64”, “x86” or “arm64” based on the machine architecture.
+* Extracts the PDFium binary (PDFium.dll) into the subfolder (x64, x86 or arm64) and consumes it to render PDF files.
```

Comment:
Specify the platform vise pdfium structure.

Reply:
Now I have specified Pdfium structure platform vice